ddtrace/contrib/aiohttp/middlewares.py

```diff
@@ -35,7 +35,11 @@ def attach_context(request):
         service = app[CONFIG_KEY]["service"]
 
         # Create a new context based on the propagated information.
-        trace_utils.activate_distributed_headers(tracer, int_config=app[CONFIG_KEY], request_headers=request.headers)
+        trace_utils.activate_distributed_headers(
```

Comment:
Does aiohttp not have a `config` object?

Reply:
It does, though I'm not sure why its integration config wasn't used in the first place. Is it fair to use that instead of having to rely on the app's stored config settings?